docs/basics/101-130-ignore.rst

```diff
@@ -0,0 +1,74 @@
+How to keep content from DataLad
+--------------------------------
+
+You have progressed quite far in the DataLad-101 course,
+and by now, you have gotten a good overview on the basics
+and *not-so-basic-anymore*\s of DataLad.
+
+Some things that you continuously did were adding, modifying,
+and saving files. In fact, the ``datalad save`` command
+is probably the most often used command in this dataset up to this
+point.
+
+This means that you have seen some of the peculiarities
+related to ``datalad save``. The most striking was that it by default
+will save the complete datasets status if one does not provide
+a path to a file change. This would result in all content
+that is either modified or untracked being saved in a single
+commit.
+This is the reason why a ``datalad run`` requires a clean
+dataset: The ``datalad save`` that a ``datalad run`` ends with
+internally should only save changes that can be attributed to
+the command that was run, and not changes that existed already
+but were yet unsaved.
+Alternatively, you learned that one can specify the ``--explicit``
```

Comment:
See my thoughts on this in #88, I think this "it-is-OK-to-be-messy" idea should appear as late as anyhow possible.